peripherals/fsmc/fsmc_wfdis.yaml

```diff
@@ -0,0 +1,8 @@
+"FMC,FSMC":
+  "BCR?":
+    WFDIS:
+      Enabled: [0, "Write FIFO enabled"]
+      Disabled: [1, "Write FIFO disabled"]
+    CCLKEN:
+      Enabled: [1, "FMC_CLK clock output to external memory devices enabled"]
+      Disabled: [0, "FMC_CLK clock output to external memory devices disabled"]
```

Comment:
The description here is a bit misleading - the clock output is always enabled during synchronous access, and CCLKEN controls whether it is _also_ output during asynchronous access (i.e. always).